core-plugins/docs/File-batchsource.md

```diff
@@ -37,6 +37,14 @@ JSON - is not supported, user has to manually provide the output schema.
 
 **Skip Header:** Whether to skip the first line of each file. Supported formats are 'text', 'csv', 'tsv', 'delimited'.
 
+**Enable Quoted Values** Whether to treat content between quotes as a value. This value will only be used if the format
+is 'csv', 'tsv' or 'delimited'. For example, if this is set to true, a line that looks like `1, "a, b, c"` will output two fields.
+The first field will have '1' as its value and the second will have `a, b, c` as its value. The quote characters will be trimmed.
+The newline delimiter cannot be within quotes.
+
+It also assumes the quotes are well enclosed. The left quote will match the first following quote right before the delimiter. If there is an
+unenclosed quote, an exception will be thrown.
```

Comment:
nit: many of our users don't know what an exception is. an exception will be thrown -> an error will occur.